server/integration/endpoint/src/test/java/org/infinispan/server/endpoint/RestConnectorTestCase.java

```diff
@@ -0,0 +1,66 @@
+package org.infinispan.server.endpoint;
+
+import static org.infinispan.server.endpoint.subsystem.ModelKeys.REST_CONNECTOR;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.OP;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.OP_ADDR;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.OUTCOME;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.READ_RESOURCE_OPERATION;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.RESULT;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.SUCCESS;
+import static org.junit.Assert.assertEquals;
+
+import java.io.IOException;
+
+import org.infinispan.server.endpoint.subsystem.EndpointExtension;
+import org.infinispan.server.endpoint.subsystem.ModelKeys;
+import org.jboss.as.clustering.infinispan.subsystem.InfinispanSubsystemDependenciesInitialization;
+import org.jboss.as.controller.PathAddress;
+import org.jboss.as.subsystem.test.AbstractSubsystemTest;
+import org.jboss.as.subsystem.test.KernelServices;
+import org.jboss.dmr.ModelNode;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * @since 9.2
+ */
+public class RestConnectorTestCase extends AbstractSubsystemTest {
```

Comment:
I must admit I feel like there should be a better way to do this, but I am fine with it

Reply:
Suggestions welcomed :)